src/api/__tests__/production-api-client.test.ts

```diff
@@ -62,6 +70,38 @@ describe( '[ProductionApiClient]', () => {
 			expect( lastRequest.requestHeaders[ fakeNonceHeaderName ] ).toEqual( fakeNonce );
 		} );
 
+		test( 'Fetches the data from cache if available and not expired', async () => {
+			const apiClient = createProductionApiClient();
+
+			localStorage.setItem( fakeCacheKey, JSON.stringify( fakeReportingConfig ) );
+			localStorage.setItem( 'cacheExpiry', ( Date.now() + 10000 ).toString() );
+
+			const reportingConfig = await apiClient.loadReportingConfig();
+
+			expect( reportingConfig ).toEqual( fakeReportingConfig );
+
+			const requests = ( server.pretender as Server & { handledRequests: any[] } ).handledRequests;
+			expect( requests.length ).toEqual( 0 );
+		} );
+
+		test( 'Fetches the reporting config again if cache has expired', async () => {
+			const apiClient = createProductionApiClient();
+
+			localStorage.setItem( fakeCacheKey, JSON.stringify( fakeReportingConfig ) );
```

Comment:
**Suggestion (non-blocking)**: Ooo, what if we made this something that was different from the one returned by the fake server, so we can be sure that we do in fact make the request and are not falling back on the cache?

Reply:
Done! I've updated it in [fc82b02](https://github.com/Automattic/bugomattic/pull/47/commits/fc82b029310534d17d908047e8f135dad061337e). Thanks for checking!